App.js

```diff
@@ -8,60 +8,66 @@
 
 import React from 'react';
 import {
-  SafeAreaView,
   StyleSheet,
-  ScrollView,
   View,
-  Text,
-  StatusBar,
   PermissionsAndroid,
   ImageBackground,
 } from 'react-native';
-import Geolocation from 'react-native-geolocation-service';
-import Weather from './components/Weather.js';
+import bgImage from './assets/background-images/sky.jpg';
+import Weather from './components/Weather';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  background: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  baseText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 18,
+  },
+});
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      hasLocationPermission: false,
+    };
   }
-  state = {
-    hasLocationPermission: false,
-  };
 
-  _requestLocationPermission = async () => {
-    try {
-      const granted = await PermissionsAndroid.request(
-        PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
-        {
-          title: 'Hey! We need your permission to use your location',
-          message: 'Cool weather App needs access to your location ',
-          buttonNeutral: 'Ask Me Later',
-          buttonNegative: 'Cancel',
-          buttonPositive: 'OK',
-        },
-      );
-      if (granted === PermissionsAndroid.RESULTS.GRANTED) {
-        console.log('You can use the location');
-        this.setState({hasLocationPermission: true});
-      } else {
-        console.log('Location permission denied');
-        this.setState({hasLocationPermission: false});
-      }
-    } catch (err) {
-      console.warn(err);
-      this.setState({hasLocationPermission: false});
+  requestLocationPermission = async () => {
+    const granted = await PermissionsAndroid.request(
```

Comment:
I think it should be named accessFineLocation or something like that because the current name doesn't describe what it really is.  
```suggestion
    const accessFineLocation = await PermissionsAndroid.request(
```